src/plugins/mmapstorage/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+include (LibAddMacros)
+
+if (DEPENDENCY_PHASE)
+	set (plugin mmapstorage)
```

Comment:
only used once, does not seem to be useful?

Reply:
Removed.